src/main/java/com/cinema/model/Role.java

```diff
@@ -0,0 +1,44 @@
+package com.cinema.model;
+
+import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToMany;
+import javax.validation.constraints.NotNull;
+
+@Entity
+public class Role {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @NotNull
+    private String roleName;
+    @ManyToMany
+    private List<User> users;
```

Comment:
why do we add users here? should role know smth about users? or maybe user should has role?